### DISCUSSION TITLE
How to update to release 3.6.0?

### DISCUSSION THREAD

**Discussion by rei-vilo** [Moddable-OpenSource/moddable#1010] [Q&A] [answered]
On a MacBook  with Apple ARM processor, perform a clean install.

---

I have performed the update of the Moddable SDK as per the [updating procedure](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/Moddable%20SDK%20-%20Getting%20Started.md#updating), including deleting the folders and building the tools again.

However, the Node-RED MCU still calls previous version `3.5.0` and the [Button does not send timestamp](https://github.com/Moddable-OpenSource/moddable/discussions/new?category=q-a#top) #68.

```
Starting build process...
Host system check: Darwin Kernel Version 22.2.0: Fri Nov 11 02:04:44 PST 2022; root:xnu-8792.61.2~4/RELEASE_ARM64_T8103
MCU Build system check: p1.1.2-post + #08c095c @ m3.5.0-0-gf9a0787
```

How to force the installation of release `3.6.0` of the Moddable SDK? Am I missing something?

Thank you!

**Comment by rei-vilo** [comment, marked as answer]
For the other thread:

> I deleted the `$MODDABLE` folder and performed a clean [install](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/Moddable%20SDK%20-%20Getting%20Started.md#installing).
> 
> Now `git describe --abbrev=7 --always  --long` shows `3.6.0-55-g5569674`.

**Reply by phoddie** [reply]
Glad you got this sorted out. It really shouldn't require a clean install, so something broke down. If you have any insight into what that might be interested so we can try to improve. Thank you.
